### ISSUE TITLE
How to unban an IP address?

### ISSUE THREAD

**Issue by sankaet** [yourabi/django-banish#16]
Hey guys, is there a way to unban an ip address?

**Comment by groovecoder** [comment]
You can delete the banishment object in the django admin UI, right?

**Comment by Anton-Shutik** [comment]
@groovecoder right, but you also need to reset a cache.

**Comment by groovecoder** [comment]
@Anton-Shutik there's [a `post_save` signal handler on `Banishment` objects that should update the cache ... at least for `ip-address` bans](https://github.com/groovecoder/django-banish/blob/master/banish/models.py#L74-77) Is it not working? I just tested this locally and it seemed to work fine.

**Comment by Anton-Shutik** [comment]
@groovecoder Yeah, but will it remove banishment from cache when deleting Banishment model instance ? So, for instance I have ip-address banned '42.42.42.42'. Will it be unbanned after just deleting Banishment object from admin ?

**Comment by groovecoder** [comment]
@Anton-Shutik that's exactly how it worked for me. As soon as I deleted the banishment object from the admin site, I was able to get back to my site from the IP address. At the very least, you should verify this and close this issue if it already works.

**Comment by groovecoder** [comment]
Anyone been able to reproduce this bug? Or can we close it?
